test/credentials_test.dart

```diff
@@ -52,55 +52,120 @@ Future<void> main([List<String>? args]) async {
     final authProvider = EmailPasswordAuthProvider(app);
-    String emailPrefix = generateRandomString(5);
-    String username = "realm_tests_do_autoverify$emailPrefix@bar.com";
+    String username = "${generateRandomString(5)}@bar.com";
     String password = "SWV23R#@T#VFQDV";
     await authProvider.registerUser(username, password);
     expect(() async {
       await authProvider.registerUser(username, password);
     }, throws<RealmException>("name already in use"));
-  });
+  }, appName: "autoConfirm");
 
   baasTest('Email/Password - register user with weak/empty password throws', (configuration) async {
     final app = App(configuration);
     final authProvider = EmailPasswordAuthProvider(app);
-    String emailPrefix = generateRandomString(5);
-    String username = "realm_tests_do_autoverify$emailPrefix@bar.com";
+    String username = "${generateRandomString(5)}@bar.com";
     expect(() async {
       await authProvider.registerUser(username, "pwd");
     }, throws<RealmException>("password must be between 6 and 128 characters"));
     expect(() async {
       await authProvider.registerUser(username, "");
     }, throws<RealmException>("password must be between 6 and 128 characters"));
-  });
+  }, appName: "autoConfirm");
 
   baasTest('Email/Password - register user with empty email throws', (configuration) async {
     final app = App(configuration);
     final authProvider = EmailPasswordAuthProvider(app);
     expect(() async {
       await authProvider.registerUser("", "password");
     }, throws<RealmException>("email invalid"));
+  }, appName: "autoConfirm");
+
+  baasTest('Email/Password - confirm user token expired', (configuration) async {
+    final app = App(configuration);
+    final authProvider = EmailPasswordAuthProvider(app);
+    String username = "${generateRandomString(5)}@hotmail.com";
+    String password = "SWV23R#@T#VFQDV";
+    await authProvider.registerUser(username, password);
+    expect(() async {
+      await authProvider.confirmUser(
+          "0e6340a446e68fe02a1af1b53c34d5f630b601ebf807d73d10a7fed5c2e996d87d04a683030377ac6058824d8555b24c1417de79019b40f1299aada7ef37fddc",
+          "6268f7dd73fafea76b730fc9");
+    }, throws<RealmException>("userpass token is expired or invalid"));
+  }, appName: "emailConfirm");
+
+  baasTest('Email/Password - confirm user token invalid', (configuration) async {
+    final app = App(configuration);
+    final authProvider = EmailPasswordAuthProvider(app);
+    String username = "${generateRandomString(5)}@hotmail.com";
+    String password = "SWV23R#@T#VFQDV";
+    await authProvider.registerUser(username, password);
+    expect(() async {
+      await authProvider.confirmUser("abc", "123");
+    }, throws<RealmException>("invalid token data"));
+  }, appName: "emailConfirm");
+
+  group("Email/Password - confirm user - manual tests", () {
+    // The tests in this group are for manual testing, since they require interaction with mail box.
```

Comment:
could we move the comment to be on the group call. Also could we rename all manual tests to include `Manual` in the name

Reply:
OK